modules/tinymce/src/core/main/ts/newline/NewLineAction.ts

```diff
@@ -46,6 +46,20 @@ const inBlock = (blockName: string, requiredState: boolean) => (editor: Editor,
   return state === requiredState;
 };
 
+const inCefPreBlock = (editor: Editor) => {
```

Comment:
Ideally, we would just have isCefBlock and isPre as two separate predicates that get AND-ed together, but I was aware that I originally wanted to reuse calculated information about what the block was. Looking at the final result, however, it seemed all of that changed, and now they should be two predicates again 

- [ ] separate into two predicates.